articles/active-directory/develop/quickstart-v2-javascript.md

```diff
@@ -77,10 +77,10 @@ Select the option that's suitable to your development environment:
 #### Step 3: Configure your JavaScript app
 
 > [!div renderon="docs"]
-> In the *JavaScriptSPA* folder, edit *authConfig.js*, and set the `clientID` and `authority` values under `msalConfig`.
+> In the *JavaScriptSPA* folder, edit *authConfig.js* file to set the `clientID`, `authority` and `redirectUri` for the `msalConfig` object.
 
 > [!div class="sxs-lookup" renderon="portal"]
```

Comment:
No longer needed since Azure portal quickstarts now come ready to run with user's app properties configured in the code. @hamiltonha